### PR TITLE
Added labels to Worf CLI arguments

### PR DESCRIPTION
**Pull request by StevenLeRoux**
Allow to pass labels as arguments to create WRITE tokens that carry fixed labels inside the token.

### CODE REVIEW

File: warp10/src/main/java/io/warp10/worf/WorfCLI.java [hunk 1]
```diff
@@ -161,6 +168,21 @@ public int execute(String[] args) {
           appName = cmd.getOptionValue(APPNAME);
         }
 
+        if (cmd.hasOption(LABELS)) try {
```
Comment (hbs) [inline warp10/src/main/java/io/warp10/worf/WorfCLI.java]: Systematically use { } after an if

File: warp10/src/main/java/io/warp10/worf/WorfCLI.java [hunk 2]
```diff
@@ -161,6 +168,21 @@ public int execute(String[] args) {
...
+            lbs = cmd.getOptionValue(LABELS);
+            String[] array = lbs.split(" ");
+            if (array.length % 2 != 0) {
+                throw new WorfException("Odd number of values provided as labels ");
+            }
+            for (int i = 0; i <= array.length - 1; i = +2) {
```
Comment (hbs) [inline warp10/src/main/java/io/warp10/worf/WorfCLI.java]: use i < array.length

File: warp10/src/main/java/io/warp10/worf/WorfCLI.java [hunk 3]
```diff
@@ -101,9 +104,12 @@ public int execute(String[] args) {
       String producerUID = null;
```
Comment (hbs) [inline warp10/src/main/java/io/warp10/worf/WorfCLI.java]: use two spaces as indent

File: warp10/src/main/java/io/warp10/worf/WorfCLI.java (hunk 2 above)
Comment (slambour) [inline warp10/src/main/java/io/warp10/worf/WorfCLI.java]: Infinite loop when labels > 2

Comment (slambour) [inline warp10/src/main/java/io/warp10/worf/WorfCLI.java]: Uses the ingress labels input format for Worf (xbeeId=XBee_40670F0D,moteId=53)
Spaces are allowed has value

**Comment by StevenLeRoux** [comment]
PR updated using ingress input format

**Comment by d33d33** [comment]
Ping

**Pull request merged**